docs/spec/v1beta1/imagerepositories.md

```diff
@@ -200,6 +205,13 @@ To grant access to all namespaces, an empty `matchLabels` must be provided:
       - matchLabels: {}
 ```
 
+### Exclude Tags
+
+To exclude certain tags, the `spec.exclusionList` field can be used to specify a list of regex expressions.
+Any tags that match any of the regex expressions, will be excluded from the final tag list.
+If `spec.exclusionList` is empty, by default the regex `"^.*\\.sig$"` is used to exclude all tags ending with
+`.sig`, since these are [Cosign](https://github.com/sigstore/cosign) generated objects and not valid images.
```

Comment:
```suggestion
`.sig`, since these are [Cosign](https://github.com/sigstore/cosign) generated objects and not container images who can be deployed on a cluster.
```